libs/shared/src/dto/subscriber/subscriber.dto.ts

```diff
@@ -36,7 +36,6 @@ export interface ISubscriberPayload {
   avatar?: string;
   locale?: string;
   data?: SubscriberCustomData;
-  [key: string]: string | string[] | boolean | number | SubscriberCustomData | undefined;
```

Comment:
Is this safe to do?

Reply:
Yes, we support custom property in the data field only.

`data?: SubscriberCustomData`